common/logger/src/kv.rs

```diff
@@ -1,11 +1,19 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-//! Key-Value definitions
+//! Key-Value definitions for macros
+//!
+//! Example:
+//! ```
+//! info!(
+//!   key = value
+//! );
+//! ```
```

Comment:
You're going to have to tweak this or turn this into a true doc test that can be compiled and run since its currently failing to run as a test in CI.